src/components/focus-trap/FocusTrap.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import Button from '../button';
+
+// @ts-ignore JS import
+import FocusTrap from './FocusTrap';
+import notes from './FocusTrap.stories.md';
+
+export const WithFocusableChildren = () => {
+    const [isVisible, setIsVisible] = React.useState(false);
+    return (
+        <div>
+            {isVisible && (
+                <FocusTrap style={{ border: '1px solid black', padding: '20px' }}>
+                    <p>
+                        <a href="/#">focusable el</a>
```

Comment:
Previously it is `<a href="#">focusable el</a>` and the href value is not recognized as valid value. Updated the href value to "/#".